chain/client/client.go

```diff
@@ -74,13 +75,30 @@ func LatestTxsRequestHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 
 		// TODO: (1) Sort result in desc order after tendermint/tendermint:#4253 is released
 		// TODO: (2) Perform binary search on 'tx.height>?' to optimize the performance
-		searchResult, err := utils.QueryTxsByEvents(cliCtx, []string{"tx.height>0"}, page, limit)
+
+		// Get all latest txs
```

Comment:
Add comment why we do this. Also reference to this PR number so we can go back here.

Reply:
@prin-r After this is landed and deployed. You can happily do wireup all txs page.